2025-04-15 01:02:32,579 [INFO] [my_app_logger] [app.py:22] - Médicaments rechargés pour génération du calendrier.
2025-04-15 01:02:32,580 [INFO] [my_app_logger] [app.py:40] - Médicaments rechargés pour l'obtention des médicaments.
2025-04-15 01:02:32,581 [DEBUG] [my_app_logger] [app.py:26] - Date de départ du calendrier reçue : 2025-04-14
2025-04-15 01:02:32,581 [DEBUG] [my_app_logger] [function.py:40] - Nombre de jours à traiter : 14
2025-04-15 01:02:32,581 [DEBUG] [my_app_logger] [function.py:41] - Date de début : 2025-04-14
2025-04-15 01:02:32,581 [DEBUG] [my_app_logger] [function.py:42] - Date de fin : 2025-04-30
2025-04-15 01:02:32,581 [DEBUG] [my_app_logger] [function.py:43] - Lundi de début : 2025-04-14
2025-04-15 01:02:32,622 [INFO] [my_app_logger] [app.py:29] - Calendrier généré avec succès.
2025-04-15 01:02:32,842 [INFO] [my_app_logger] [app.py:40] - Médicaments rechargés pour l'obtention des médicaments.
2025-04-15 01:02:32,888 [INFO] [my_app_logger] [app.py:22] - Médicaments rechargés pour génération du calendrier.
2025-04-15 01:02:32,889 [DEBUG] [my_app_logger] [app.py:26] - Date de départ du calendrier reçue : 2025-04-14
2025-04-15 01:02:32,889 [DEBUG] [my_app_logger] [function.py:40] - Nombre de jours à traiter : 14
2025-04-15 01:02:32,889 [DEBUG] [my_app_logger] [function.py:41] - Date de début : 2025-04-14
2025-04-15 01:02:32,889 [DEBUG] [my_app_logger] [function.py:42] - Date de fin : 2025-04-30
2025-04-15 01:02:32,889 [DEBUG] [my_app_logger] [function.py:43] - Lundi de début : 2025-04-14
2025-04-15 01:02:32,928 [INFO] [my_app_logger] [app.py:29] - Calendrier généré avec succès.
2025-04-15 01:05:35,394 [INFO] [my_app_logger] [app.py:40] - Médicaments rechargés pour l'obtention des médicaments.
2025-04-15 01:05:35,663 [INFO] [my_app_logger] [app.py:40] - Médicaments rechargés pour l'obtention des médicaments.
2025-04-15 01:05:55,521 [INFO] [my_app_logger] [app.py:22] - Médicaments rechargés pour génération du calendrier.
2025-04-15 01:05:55,521 [DEBUG] [my_app_logger] [app.py:26] - Date de départ du calendrier reçue : 2025-04-14
2025-04-15 01:05:55,521 [DEBUG] [my_app_logger] [function.py:40] - Nombre de jours à traiter : 14
2025-04-15 01:05:55,522 [DEBUG] [my_app_logger] [function.py:41] - Date de début : 2025-04-14
2025-04-15 01:05:55,522 [DEBUG] [my_app_logger] [function.py:42] - Date de fin : 2025-04-30
2025-04-15 01:05:55,522 [DEBUG] [my_app_logger] [function.py:43] - Lundi de début : 2025-04-14
2025-04-15 01:05:55,566 [INFO] [my_app_logger] [app.py:29] - Calendrier généré avec succès.
2025-04-15 01:05:55,877 [INFO] [my_app_logger] [app.py:22] - Médicaments rechargés pour génération du calendrier.
2025-04-15 01:05:55,877 [DEBUG] [my_app_logger] [app.py:26] - Date de départ du calendrier reçue : 2025-04-14
2025-04-15 01:05:55,877 [DEBUG] [my_app_logger] [function.py:40] - Nombre de jours à traiter : 14
2025-04-15 01:05:55,877 [DEBUG] [my_app_logger] [function.py:41] - Date de début : 2025-04-14
2025-04-15 01:05:55,877 [DEBUG] [my_app_logger] [function.py:42] - Date de fin : 2025-04-30
2025-04-15 01:05:55,877 [DEBUG] [my_app_logger] [function.py:43] - Lundi de début : 2025-04-14
2025-04-15 01:05:55,918 [INFO] [my_app_logger] [app.py:29] - Calendrier généré avec succès.
2025-04-16 10:38:04,732 [INFO] [my_app_logger] [app.py:22] - Médicaments rechargés pour génération du calendrier.
2025-04-16 10:38:04,736 [INFO] [my_app_logger] [app.py:40] - Médicaments rechargés pour l'obtention des médicaments.
2025-04-16 10:38:04,744 [DEBUG] [my_app_logger] [app.py:26] - Date de départ du calendrier reçue : 2025-04-16
2025-04-16 10:38:04,745 [DEBUG] [my_app_logger] [function.py:40] - Nombre de jours à traiter : 14
2025-04-16 10:38:04,745 [DEBUG] [my_app_logger] [function.py:41] - Date de début : 2025-04-16
2025-04-16 10:38:04,745 [DEBUG] [my_app_logger] [function.py:42] - Date de fin : 2025-04-30
2025-04-16 10:38:04,745 [DEBUG] [my_app_logger] [function.py:43] - Lundi de début : 2025-04-14
2025-04-16 10:38:04,841 [INFO] [my_app_logger] [app.py:29] - Calendrier généré avec succès.
2025-04-16 10:38:04,983 [INFO] [my_app_logger] [app.py:40] - Médicaments rechargés pour l'obtention des médicaments.
2025-04-16 10:38:05,076 [INFO] [my_app_logger] [app.py:22] - Médicaments rechargés pour génération du calendrier.
2025-04-16 10:38:05,077 [DEBUG] [my_app_logger] [app.py:26] - Date de départ du calendrier reçue : 2025-04-16
2025-04-16 10:38:05,077 [DEBUG] [my_app_logger] [function.py:40] - Nombre de jours à traiter : 14
2025-04-16 10:38:05,077 [DEBUG] [my_app_logger] [function.py:41] - Date de début : 2025-04-16
2025-04-16 10:38:05,077 [DEBUG] [my_app_logger] [function.py:42] - Date de fin : 2025-04-30
2025-04-16 10:38:05,077 [DEBUG] [my_app_logger] [function.py:43] - Lundi de début : 2025-04-14
2025-04-16 10:38:05,145 [INFO] [my_app_logger] [app.py:29] - Calendrier généré avec succès.
2025-04-16 10:42:43,316 [INFO] [my_app_logger] [app.py:22] - Médicaments rechargés pour génération du calendrier.
2025-04-16 10:42:43,316 [DEBUG] [my_app_logger] [app.py:26] - Date de départ du calendrier reçue : 2025-04-16
2025-04-16 10:42:43,316 [DEBUG] [my_app_logger] [function.py:40] - Nombre de jours à traiter : 14
2025-04-16 10:42:43,316 [DEBUG] [my_app_logger] [function.py:41] - Date de début : 2025-04-16
2025-04-16 10:42:43,317 [DEBUG] [my_app_logger] [function.py:42] - Date de fin : 2025-04-30
2025-04-16 10:42:43,317 [DEBUG] [my_app_logger] [function.py:43] - Lundi de début : 2025-04-14
2025-04-16 10:42:43,334 [INFO] [my_app_logger] [app.py:40] - Médicaments rechargés pour l'obtention des médicaments.
2025-04-16 10:42:43,421 [INFO] [my_app_logger] [app.py:29] - Calendrier généré avec succès.
2025-04-16 10:42:43,575 [INFO] [my_app_logger] [app.py:40] - Médicaments rechargés pour l'obtention des médicaments.
2025-04-16 10:42:43,707 [INFO] [my_app_logger] [app.py:22] - Médicaments rechargés pour génération du calendrier.
2025-04-16 10:42:43,707 [DEBUG] [my_app_logger] [app.py:26] - Date de départ du calendrier reçue : 2025-04-16
2025-04-16 10:42:43,707 [DEBUG] [my_app_logger] [function.py:40] - Nombre de jours à traiter : 14
2025-04-16 10:42:43,707 [DEBUG] [my_app_logger] [function.py:41] - Date de début : 2025-04-16
2025-04-16 10:42:43,708 [DEBUG] [my_app_logger] [function.py:42] - Date de fin : 2025-04-30
2025-04-16 10:42:43,708 [DEBUG] [my_app_logger] [function.py:43] - Lundi de début : 2025-04-14
2025-04-16 10:42:43,794 [INFO] [my_app_logger] [app.py:29] - Calendrier généré avec succès.
2025-04-16 10:44:59,889 [INFO] [my_app_logger] [app.py:22] - Médicaments rechargés pour génération du calendrier.
2025-04-16 10:44:59,890 [DEBUG] [my_app_logger] [app.py:26] - Date de départ du calendrier reçue : 2025-04-16
2025-04-16 10:44:59,890 [DEBUG] [my_app_logger] [function.py:40] - Nombre de jours à traiter : 14
2025-04-16 10:44:59,890 [DEBUG] [my_app_logger] [function.py:41] - Date de début : 2025-04-16
2025-04-16 10:44:59,890 [DEBUG] [my_app_logger] [function.py:42] - Date de fin : 2025-04-30
2025-04-16 10:44:59,890 [DEBUG] [my_app_logger] [function.py:43] - Lundi de début : 2025-04-14
2025-04-16 10:44:59,953 [INFO] [my_app_logger] [app.py:40] - Médicaments rechargés pour l'obtention des médicaments.
2025-04-16 10:45:00,069 [INFO] [my_app_logger] [app.py:29] - Calendrier généré avec succès.
2025-04-16 10:45:00,139 [ERROR] [my_app_logger] [routes.py:31] - [FRONTEND] Médicaments récupérés : | Details: None
2025-04-16 10:45:00,224 [INFO] [my_app_logger] [app.py:40] - Médicaments rechargés pour l'obtention des médicaments.
2025-04-16 10:45:00,307 [ERROR] [my_app_logger] [routes.py:31] - [FRONTEND] Médicaments récupérés : | Details: None
2025-04-16 10:45:00,328 [INFO] [my_app_logger] [app.py:22] - Médicaments rechargés pour génération du calendrier.
2025-04-16 10:45:00,329 [DEBUG] [my_app_logger] [app.py:26] - Date de départ du calendrier reçue : 2025-04-16
2025-04-16 10:45:00,329 [DEBUG] [my_app_logger] [function.py:40] - Nombre de jours à traiter : 14
2025-04-16 10:45:00,329 [DEBUG] [my_app_logger] [function.py:41] - Date de début : 2025-04-16
2025-04-16 10:45:00,330 [DEBUG] [my_app_logger] [function.py:42] - Date de fin : 2025-04-30
2025-04-16 10:45:00,330 [DEBUG] [my_app_logger] [function.py:43] - Lundi de début : 2025-04-14
2025-04-16 10:45:00,472 [INFO] [my_app_logger] [app.py:29] - Calendrier généré avec succès.
2025-04-16 10:46:04,824 [INFO] [my_app_logger] [app.py:40] - Médicaments rechargés pour l'obtention des médicaments.
2025-04-16 10:46:04,825 [INFO] [my_app_logger] [app.py:22] - Médicaments rechargés pour génération du calendrier.
2025-04-16 10:46:04,826 [DEBUG] [my_app_logger] [app.py:26] - Date de départ du calendrier reçue : 2025-04-16
2025-04-16 10:46:04,826 [DEBUG] [my_app_logger] [function.py:40] - Nombre de jours à traiter : 14
2025-04-16 10:46:04,826 [DEBUG] [my_app_logger] [function.py:41] - Date de début : 2025-04-16
2025-04-16 10:46:04,827 [DEBUG] [my_app_logger] [function.py:42] - Date de fin : 2025-04-30
2025-04-16 10:46:04,827 [DEBUG] [my_app_logger] [function.py:43] - Lundi de début : 2025-04-14
2025-04-16 10:46:04,932 [INFO] [my_app_logger] [app.py:29] - Calendrier généré avec succès.
2025-04-16 10:46:05,064 [INFO] [my_app_logger] [app.py:40] - Médicaments rechargés pour l'obtention des médicaments.
2025-04-16 10:46:05,158 [INFO] [my_app_logger] [app.py:22] - Médicaments rechargés pour génération du calendrier.
2025-04-16 10:46:05,158 [DEBUG] [my_app_logger] [app.py:26] - Date de départ du calendrier reçue : 2025-04-16
2025-04-16 10:46:05,158 [DEBUG] [my_app_logger] [function.py:40] - Nombre de jours à traiter : 14
2025-04-16 10:46:05,158 [DEBUG] [my_app_logger] [function.py:41] - Date de début : 2025-04-16
2025-04-16 10:46:05,158 [DEBUG] [my_app_logger] [function.py:42] - Date de fin : 2025-04-30
2025-04-16 10:46:05,159 [DEBUG] [my_app_logger] [function.py:43] - Lundi de début : 2025-04-14
2025-04-16 10:46:05,183 [ERROR] [my_app_logger] [routes.py:31] - [FRONTEND] Médicaments récupérés : | Details: None
2025-04-16 10:46:05,249 [INFO] [my_app_logger] [app.py:29] - Calendrier généré avec succès.
2025-04-16 10:46:05,260 [ERROR] [my_app_logger] [routes.py:31] - [FRONTEND] Médicaments récupérés : | Details: None
2025-04-16 10:54:04,741 [INFO] [my_app_logger] [app.py:22] - Médicaments rechargés pour génération du calendrier.
2025-04-16 10:54:04,742 [INFO] [my_app_logger] [app.py:40] - Médicaments rechargés pour l'obtention des médicaments.
2025-04-16 10:54:04,748 [DEBUG] [my_app_logger] [app.py:26] - Date de départ du calendrier reçue : 2025-04-16
2025-04-16 10:54:04,749 [DEBUG] [my_app_logger] [function.py:40] - Nombre de jours à traiter : 14
2025-04-16 10:54:04,749 [DEBUG] [my_app_logger] [function.py:41] - Date de début : 2025-04-16
2025-04-16 10:54:04,749 [DEBUG] [my_app_logger] [function.py:42] - Date de fin : 2025-04-30
2025-04-16 10:54:04,749 [DEBUG] [my_app_logger] [function.py:43] - Lundi de début : 2025-04-14
2025-04-16 10:54:04,834 [INFO] [my_app_logger] [app.py:29] - Calendrier généré avec succès.
2025-04-16 10:54:04,985 [INFO] [my_app_logger] [app.py:40] - Médicaments rechargés pour l'obtention des médicaments.
2025-04-16 10:54:05,094 [INFO] [my_app_logger] [app.py:22] - Médicaments rechargés pour génération du calendrier.
2025-04-16 10:54:05,095 [DEBUG] [my_app_logger] [app.py:26] - Date de départ du calendrier reçue : 2025-04-16
2025-04-16 10:54:05,095 [DEBUG] [my_app_logger] [function.py:40] - Nombre de jours à traiter : 14
2025-04-16 10:54:05,095 [DEBUG] [my_app_logger] [function.py:41] - Date de début : 2025-04-16
2025-04-16 10:54:05,095 [DEBUG] [my_app_logger] [function.py:42] - Date de fin : 2025-04-30
2025-04-16 10:54:05,095 [DEBUG] [my_app_logger] [function.py:43] - Lundi de début : 2025-04-14
2025-04-16 10:54:05,182 [INFO] [my_app_logger] [routes.py:37] - Info du frontend : Médicaments récupérés : - None
2025-04-16 10:54:05,184 [INFO] [my_app_logger] [app.py:29] - Calendrier généré avec succès.
2025-04-16 10:54:05,397 [INFO] [my_app_logger] [routes.py:37] - Info du frontend : Médicaments récupérés : - None
2025-04-16 10:55:38,343 [ERROR] [my_app_logger] [app.py:43] - Erreur dans /get_pils
Traceback (most recent call last):
  File "c:\Users\mtx_2\Documents\Code\Medic\Medic\backend\app.py", line 38, in get_pils
    with open("pils.json", encoding="utf-8") as f:
         ~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'pils.json'
2025-04-16 10:55:38,344 [ERROR] [my_app_logger] [app.py:32] - Erreur dans /calendar
Traceback (most recent call last):
  File "c:\Users\mtx_2\Documents\Code\Medic\Medic\backend\app.py", line 20, in get_calendar
    with open("pils.json", encoding="utf-8") as f:
         ~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'pils.json'
2025-04-16 10:55:38,592 [ERROR] [my_app_logger] [app.py:43] - Erreur dans /get_pils
Traceback (most recent call last):
  File "c:\Users\mtx_2\Documents\Code\Medic\Medic\backend\app.py", line 38, in get_pils
    with open("pils.json", encoding="utf-8") as f:
         ~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'pils.json'
2025-04-16 10:55:38,680 [ERROR] [my_app_logger] [app.py:32] - Erreur dans /calendar
Traceback (most recent call last):
  File "c:\Users\mtx_2\Documents\Code\Medic\Medic\backend\app.py", line 20, in get_calendar
    with open("pils.json", encoding="utf-8") as f:
         ~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'pils.json'
2025-04-16 10:55:38,956 [INFO] [my_app_logger] [routes.py:37] - Info du frontend : Médicaments récupérés avec succès - None
2025-04-16 10:55:39,027 [ERROR] [my_app_logger] [routes.py:33] - Erreur du frontend : Erreur de récupération du calendrier : - {}
2025-04-16 10:55:39,044 [ERROR] [my_app_logger] [routes.py:33] - Erreur du frontend : Erreur de récupération du calendrier : - {}
2025-04-16 10:55:39,045 [INFO] [my_app_logger] [routes.py:37] - Info du frontend : Médicaments récupérés avec succès - None
